flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -498,6 +498,12 @@ public class KubernetesConfigOptions {
                             "Whether to enable HostNetwork mode. "
                                     + "The HostNetwork allows the pod could use the node network namespace instead of the individual pod network namespace. Please note that the JobManager service account should have the permission to update Kubernetes service.");
 
+    public static final ConfigOption<String> KUBERNETES_KUBECLIENT_USER_AGENT =
```

Comment:
Following the naming pattern `kubernetes.client.io-pool.size`, maybe `kubernetes.client.user-agent` is better.